Cargo.toml

```
@@ -17,3 +17,4 @@ fdstore = ["dep:sendfd"]
 
 [dependencies]
 sendfd = { version = "0.4", optional = true }
+libc = "^0.2.169"
```

I think this will work:

```suggestion
libc = "0.2"
```